### PR TITLE
Updates from bedrock

### PR DESCRIPTION
**Pull request by MozmarRobot**
From file changes in https://github.com/mozilla/bedrock/commit/15f0091f75ab3ef110cd18d083845d0e31eeb9a3

### CODE REVIEW

**Comment by alexgibson** [comment]
@peiying2 r? There are some strings here for welcome10.ftl that we would like to get into Pontoon as a priority. Thanks

File: configs/pontoon.toml [hunk 1]
```diff
@@ -164,8 +164,34 @@ locales = [
...
+[[paths]]
+    reference = "en/firefox/welcome/page2.ftl"
+    l10n = "{locale}/firefox/welcome/page2.ftl"
+[[paths]]
+    reference = "en/firefox/welcome/page3.ftl"
+    l10n = "{locale}/firefox/welcome/page3.ftl"
+[[paths]]
+    reference = "en/firefox/welcome/page4.ftl"
+    l10n = "{locale}/firefox/welcome/page4.ftl"
+[[paths]]
```
Comment (peiying2) [inline configs/pontoon.toml:178]: Should we add page5?

File: configs/pontoon.toml [hunk 2]
```diff
@@ -164,8 +164,34 @@ locales = [
...
+    reference = "en/firefox/welcome/page6.ftl"
+    l10n = "{locale}/firefox/welcome/page6.ftl"
+[[paths]]
+    reference = "en/firefox/welcome/page7.ftl"
+    l10n = "{locale}/firefox/welcome/page7.ftl"
+[[paths]]
+    reference = "en/firefox/welcome/page8.ftl"
+    l10n = "{locale}/firefox/welcome/page8.ftl"
+[[paths]]
```
Comment (peiying2) [inline configs/pontoon.toml:187]: Do we have page9 in the work?

File: configs/pontoon.toml [hunk 3]
```diff
@@ -164,8 +164,34 @@ locales = [
...
+    reference = "en/firefox/welcome/page10.ftl"
+    l10n = "{locale}/firefox/welcome/page10.ftl"
+    locales = [
```
Comment (peiying2) [inline configs/pontoon.toml:190]: no `de`, `fr`?

File: configs/pontoon.toml (hunk 1 above)
Comment (alexgibson) [reply configs/pontoon.toml:178]: Page 5 is hard coded and does not use an .ftl

File: configs/pontoon.toml (hunk 2 above)
Comment (alexgibson) [reply configs/pontoon.toml:187]: Same for page 9 no ftl

File: configs/pontoon.toml (hunk 3 above)
Comment (alexgibson) [reply configs/pontoon.toml:190]: Not this time no, thanks

File: configs/pontoon.toml (hunk 1 above)
Comment (peiying2) [reply configs/pontoon.toml:178]: It's in Pontoon: https://pontoon.mozilla.org/de/mozillaorg/en/firefox/welcome/page5.ftl/?string=212989

**Review by peiying2 — COMMENTED** [review]
Check page5 as it is in Pontoon: It's in Pontoon: https://pontoon.mozilla.org/de/mozillaorg/en/firefox/welcome/page5.ftl/?string=212989. Otherwise, the PR looks good.

**Comment by alexgibson** [comment]
@peiying2 it looks like welcome/5 was removed from bedrock in https://github.com/mozilla/bedrock/pull/10635

**Comment by peiying2** [comment]
I will merge this and will follow up with what happened with page5.

**Review by peiying2 — APPROVED** [review]

**Comment by peiying2** [comment]
@alexgibson somehow, this page didn't land for any languages in Pontoon.

**Comment by alexgibson** [comment]
@peiying2 i see the files if I hit the URL directly. Perhaps it’s just a cache issue? 

https://pontoon.mozilla.org/es-ES/mozillaorg/en/firefox/welcome/page10.ftl/

**Comment by alexgibson** [comment]
I see them in the index now. `page10.ftl` shows up immediately before `page1.ftl` because of the way Pontoon orders files.

**Comment by peiying2** [comment]
There was a bit delay in the sync. All is good now. But that ordering is questionable, :).

**Pull request merged**